### ISSUE TITLE
网络输出的代码错了吧？output      = torch.cat([reg_output, obj_output, cls_output], 1)

### ISSUE THREAD

**Issue by mepleleo** [bubbliiiing/yolox-pytorch#55]
复现代码是
output      = torch.cat([reg_output, obj_output, cls_output], 1)
源代码是
 output = torch.cat(
                    [reg_output, obj_output.sigmoid(), cls_output.sigmoid()], 1
                )
复现代码没加激活函数啊？

**Comment by bubbliiiing** [comment]
我可以在别的地方加呢
